test/bdd/fixtures/.env

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
suggest a subfolder for a particular composition?

Reply:
@troyronda I can't put composition in subfolder because our bddtest structure require to start one docker-compose https://github.com/hyperledger/aries-framework-go/pull/499/files#r336109199

Reply:
Should still be a subfolder. In Fabric SDK Go, I think we called it something like dockerenv

Reply:
@troyronda I will create subfolder

Reply:
@troyronda done